site/content/docs/main/release-instructions.md

```diff
@@ -159,7 +159,7 @@ What to include in a release blog:
 
 Release blog post PR:
 * Prepare a PR containing the release blog post. Read the [website guidelines][2] for more information on creating a blog post. It's usually easiest to make a copy of the most recent existing post, then replace the content as appropriate.
-* You also need to update `site/index.html` to have "Latest Release Information" contain a link to the new post.
+* You also need to update `site/content/_index.md` to have "Latest Release Information" contain a link to the new post.
```

Comment:
Should I change in other versions or just main?

Reply:
Since there are no blog posts for new releases for a while, I think only v1.11 and the main branches need this change.

Reply:
My opinion is just fixing in main is enough.